src/users/users.controller.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    Controller,
+    Get,
+    Post,
+    Put,
+    Delete,
+    Param,
+    Body,
+} from '@nestjs/common';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { UsersService } from './users.service';
+
+@Controller('users')
```

Comment:
:wrench: Let's add a small hint that this controller is not complete yet and will receive it's proper implementation in #34 

```suggestion
//TODO: #34 - Overhaul Controller methods according to use cases
@Controller('user')
```